### ISSUE TITLE
Square radio button embedded in UITableView appear rounded. Only selecting it changes to square.

### ISSUE THREAD

**Issue by TylorPin** [DavydLiu/DLRadioButton#118]
![2017-12-27-gif-00000086](https://user-images.githubusercontent.com/4278637/34377719-3ca33390-eb2d-11e7-8a37-856ac87979ad.gif)

I have some radio button for a survey module, for question with select at least one i put rounded radio button, question with select one i put square button.

As the GIF shows, the squared radio button is appearing rounded, only when i clicked on it, it turns square. Do i need to call any function to update uitableviewcell when it appear?

**Comment by DavydLiu** [comment]
I think you are reusing the same group of radio buttons for different cells. Try to put groups of radio buttons in an array and fetch them when updating table cells.

**Comment by TylerPin** [comment]
Yeah. Searched online and find out that UIButton can't change state after it is initialised. Current new method im doing is set a rounded and rectangular button and show/hide.
